packages/api/types.d.ts

```diff
@@ -258,6 +261,46 @@ declare global {
 		 */
 		quote: string
 	}
+
+	type KrakenAssetList = {
```

Comment:
same for this, this type is specific to the kraken module and should remain private imo, can you move it back to kraken? thx